src/base/static/components/form-fields/field-definitions.js

```diff
@@ -206,8 +206,8 @@ export default {
         markers={fieldConfig.content.map(item => item.marker)}
         existingGeometry={context.props.existingGeometry}
         existingGeometryStyle={context.props.existingGeometryStyle}
-        existingCollectionId={context.props.existingCollectionId}
-        existingModelId={context.props.existingModelId}
```

Comment:
In thisPR we remove the terms `collection` and `model`, replacing them with `dataset` and `place`.